site/_config.yml

```diff
@@ -32,3 +32,7 @@ gems:
   - jekyll-seo-tag
   - jekyll-avatar
   - jekyll-mentions
+
+exclude:
+  - README.md
+  - .gitignore
```

Comment:
@crispgm Yes! I mean that Jekyll should ignore it when building. See [EntryFilter](https://github.com/jekyll/jekyll/blob/b5ee87b4ca91fefe86c42526fee230a3c0294eca/lib/jekyll/entry_filter.rb#L4-L6).